CurationMarkets.sol

```diff
@@ -95,14 +98,17 @@ contract ContinuousToken is ERC20Token {
     }
 
     function withdraw(uint256 _amountToWithdraw) returns (bool) {
-        if(balances[msg.sender] - _amountToWithdraw > 0) {
+        if(_amountToWithdraw > 0 && balances[msg.sender] >= _amountToWithdraw) {
             //determine how much you can leave with.
-            uint256 reward = _amountToWithdraw/totalSupply * poolBalance; //rounding?
+            uint256 reward = _amountToWithdraw * poolBalance/totalSupply; //rounding?
```

Comment:
Because all the math in Solidity is fixed-point and rounding happens in the division first which most of the times will just mean `_amountToWithdraw/totalSupply == 0` and so `reward == 0`.

(v. https://ethereum.stackexchange.com/a/2990/5973)